<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelCelView</class>
 <widget class="QWidget" name="LevelCelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="celFrameWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="celLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="celGraphicsView">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>360</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="levelCelControlsGridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="tilLabel">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Tiles :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="13">
         <widget class="QLabel" name="tilFrameHeightLabel">
          <property name="text">
           <string>px</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QPushButton" name="nextSubtileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Next group</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="12">
         <widget class="QPushButton" name="cloneSubTileButton">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-copy"/>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="firstTileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>First group</string>
          </property>
          <property name="text">
           <string>|&lt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" colspan="2">
         <widget class="QPushButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="zoomLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Zoom:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="zoomEdit">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="previousFrameButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Previous frame</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="framesLabel">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Frames :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="11">
         <widget class="QLabel" name="playDelayLabel">
          <property name="text">
           <string>ms</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="6">
         <widget class="QLineEdit" name="frameIndexEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Current frame</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QLineEdit" name="subtileNumberEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of groups</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="zoomOutButton">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="12">
         <widget class="QPushButton" name="cloneTileButton">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-copy"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="minLabel">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Sub-tiles :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="13">
         <widget class="QLabel" name="celFrameHeightLabel">
          <property name="text">
           <string>px</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="12">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="10">
         <widget class="QLineEdit" name="playDelayEdit">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Current playback delay</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QLineEdit" name="frameNumberEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of frames</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="10">
         <widget class="QPushButton" name="lastFrameButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Last frame</string>
          </property>
          <property name="text">
           <string>&gt;|</string>
          </property>
         </widget>
        </item>
        <item row="1" column="11">
         <widget class="QPushButton" name="addTileButton">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="minSlashLabel">
          <property name="minimumSize">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>8</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="zoomInButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8" colspan="2">
         <widget class="QPushButton" name="playButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLineEdit" name="subtileIndexEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Current group</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="11">
         <widget class="QPushButton" name="addSubTileButton">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="firstSubtileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>First group</string>
          </property>
          <property name="text">
           <string>|&lt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="previousSubtileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Previous group</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="13">
         <widget class="QLabel" name="minFrameHeightLabel">
          <property name="text">
           <string>px</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="slashLabel">
          <property name="minimumSize">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>8</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLabel" name="tilSlashLabel">
          <property name="minimumSize">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>8</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="tileIndexEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Current group</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="slashLabel_3">
          <property name="minimumSize">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>8</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="firstFrameButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>First frame</string>
          </property>
          <property name="text">
           <string>|&lt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QLineEdit" name="tileNumberEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of groups</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QPushButton" name="lastSubtileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Last group</string>
          </property>
          <property name="text">
           <string>&gt;|</string>
          </property>
         </widget>
        </item>
        <item row="0" column="13">
         <widget class="QComboBox" name="playComboBox">
          <item>
           <property name="text">
            <string>Caves</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crypt</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QPushButton" name="lastTileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Last group</string>
          </property>
          <property name="text">
           <string>&gt;|</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QPushButton" name="nextTileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Next group</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="QPushButton" name="nextFrameButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Next frame</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="previousTileButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Previous group</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tilesTabs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
